.github/workflows/ci.yml

```diff
@@ -10,10 +10,10 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 16
-          cache: 'yarn'
+          cache: 'npm'
 
-      - name: Yarn Install
-        run: yarn install --immutable
+      - name: NPM Install
+        run: npm install --immutable
```

Comment:
I'd do `npm ci` here.

Reply:
yep, sorry forgot

Reply:
seemed to build alright with install though